RetailCoder.VBE/Settings/GeneralConfigProvider.cs

```diff
@@ -53,20 +53,21 @@ private void CheckForEventsToRaise(GeneralSettings other)
         protected virtual void OnLanguageChanged(EventArgs e)
         {
             var handler = LanguageChanged;
-            if (handler != null)
-            {
-                handler(this, e);
-            }
+            LanguageChanged?.Invoke(this, e);
```

Comment:
this isn't semantically equivalent. The local copy is a protection against multithreading issues like `LanguageChanged` being removed after the invocation of the method but before any execution...

I'm not quite sure as to how much atomicity is guaranteed by `?.` over an explicit null-check on a local variable (which is guaranteed to be atomic)